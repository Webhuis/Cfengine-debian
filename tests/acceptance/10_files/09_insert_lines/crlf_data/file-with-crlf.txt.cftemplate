alpha
Inserted line
beta
[%CFEngine BEGIN %]
Inserted block
Block line 1
Block line 2
[%CFEngine END %]
charlie
