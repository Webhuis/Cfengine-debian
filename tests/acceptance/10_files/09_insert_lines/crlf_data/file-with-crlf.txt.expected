alpha
Inserted line
beta
Inserted block
Block line 1
Block line 2
charlie
